options/locale/locale_en-US.ini

```diff
@@ -61,103 +61,103 @@ cancel = Cancel
-server_service_title = Server and Other Services Settings
-offline_mode = Enable Offline Mode
-offline_mode_popup = Disable CDN so all resource files will be served locally.
-disable_gravatar = Disable Gravatar Service
-disable_gravatar_popup = Disable Gravatar and custom sources. All avatars must be uploaded by users or the default avatar will be used.
-federated_avatar_lookup = Enable Federated Avatars Lookup
+smtp_from = Send Email As
+smtp_from_helper = Email address Gitea will use. Enter a plain email address or use the "Name" <email@example.com> format.
+mailer_user = SMTP Username
+mailer_password = SMTP Password
+register_confirm = Require Email Confirmation to Register
+mail_notify = Enable Email Notifications
+server_service_title = Server and Third-Party Service Settings
+offline_mode = Enable Local Mode
+offline_mode_popup = Disable third-party content delivery networks and serve all resources locally.
+disable_gravatar = Disable Gravatar
+disable_gravatar_popup = Disable Gravatar and third-party avatar sources. A default avatar will be used unless a user locally uploads an avatar.
+federated_avatar_lookup = Enable Federated Avatars
 federated_avatar_lookup_popup = Enable federated avatar lookup using Libravatar.
-disable_registration = Disable Self-registration
-disable_registration_popup = "Disable self-registration; only admins will be able to create accounts."
+disable_registration = Disable Self-Registration
+disable_registration_popup = Disable user self-registration. Only administrators will be able to create new user accounts.
 openid_signin = Enable OpenID Sign-In
-openid_signin_popup = Enable user login via OpenID
-openid_signup = Enable OpenID Self-registration
-openid_signup_popup = Enable OpenID based Self-registration
-enable_captcha = Enable Captcha
+openid_signin_popup = Enable user sign-in via OpenID.
+openid_signup = Enable OpenID Self-Registration
+openid_signup_popup = Enable OpenID-based user self-registration.
+enable_captcha = Enable CAPTCHA
 enable_captcha_popup = Require a CAPTCHA for user self-registration.
-require_sign_in_view = Enable Require Sign In to View Pages
-require_sign_in_view_popup = "Only signed in users can view pages; visitors will only be able to see the sign in and up pages."
-admin_setting_desc = You do not need to create an admin account right now. The first user who registers on the site will gain admin access automatically.
-admin_title = Admin Account Settings
+require_sign_in_view = Require Sign-In to View Pages
+require_sign_in_view_popup = Limit page access to signed-in users. Visitors will only see the 'sign in' and registration pages.
+admin_setting_desc = Creating an administrator account is optional. The first registered user will automatically become an administrator.
+admin_title = Administrator Account Settings
 admin_name = Username
 admin_password = Password
 confirm_password = Confirm Password
-admin_email = Admin Email
+admin_email = Administrator Email Address
```

Comment:
i'd drop the "Administrator" here as you already specified the context in `admin_title` and `admin_name` doesn't contain it either.

Reply:
@funkyfuture how about using `Administrator Username` and then `Email Address`? That would solve the visual imbalance between label lengths, while still emphasizing the fact that this is an administrator account. I'm all for removing repetition, but I think this account should be emphasized a little bit.

Reply:
there's a heading above and an introductary sentence that makes the scope very clear. no need for further emphasis - i actually felt very annoyed by the redundance at this point. your proposal is actually imbalancing the width. if you're keen on that 'Confirm' should suffice.

Reply:
I'm all for at least mentioning 'Administrator' once in the bold labels. It's hard to label and describe options in a foolproof way. I want to accommodate at least a minimum level of reading incomprehension. Will update my branch in a moment so you can see the change; it looks robust.